Resources/Prototypes/Traits/disabilities.yml

```diff
@@ -22,3 +22,19 @@
   name: trait-pacifist-name
   components:
     - type: Pacifist
+
+- type: trait
+  id: Tourette
+  name: trait-tourette-name
+  description: trait-tourette-desc
+  components:
+    - type: TouretteSyndrome
+      timeBetweenIncidents: 60, 400
+      touretteSymptoms:
+        - Scream
+        - Phrase
+        - DropItemInHand
+        - UseItemInHand
+        - Jittering
+      tourettePhrases: ["Ccloooown", "fuck", "woof-woof", "Grrr Grrr", "HrrHrHrr", "khaaaaa", "pizza pizzza", "you dumb", "banana peel", "hit meee"] # need locale optimization
```

Comment:
im not sure about this one mate

Reply:
Yes, i agree, but idk how to make it available for locale